forge-gui/res/cardsfolder/upcoming/valgavoth_terror_eater.txt

```diff
@@ -0,0 +1,11 @@
+Name:Valgavoth, Terror Eater
+ManaCost:6 B B B
+Types:Legendary Creature Elder Demon
+PT:9/9
+K:Flying
+K:Lifelink
+K:Ward:Sac<3/Permanent.nonLand/nonland permanent>
+R:Event$ Moved | ActiveZones$ Battlefield | Destination$ Graveyard | ValidCard$ Card.nonToken+YouDontCtrl+OppOwn | ReplaceWith$ Exile | Description$ If a card you didn't control would be put into an opponent's graveyard from anywhere, exile it instead.

```

Comment:
you need to test this
it might need ValidLKI$ instead

Reply:
I tested Valgavoth, seemed to exile cards alright and when my stolen card died, it went normally to graveyard.